src/Query/index.ts

```diff
@@ -10,6 +10,49 @@ function processPart(part: string): string[] {
   return tokenize(part, false);
 }
 
+const BURIED_PROPERTIES = ['realm'];
+const BURIED_FACTOR = 0.8;
+
+//check type of parts
+function constructAgg(parts: any[]): object[] {
+  const newParts: any[] = [];
+  for (var part of parts) {
+    //push to two compounds for each part to the new array
+    newParts.push(
+      //if given query matches a "part" result not in BURIED_PROPERTY(ex: Realm) docs, score remains unaffected
+      {
+        compound: {
+          must: [part],
+          mustNot: [
+            {
+              text: {
+                query: BURIED_PROPERTIES,
+                path: 'searchProperty',
```

Comment:
Just curious: I'm assuming `searchProperty` is easier to use (and more convenient right now) than using the `facets` field?

Reply:
Yes, also because most search-staging documents have a `null` `facets` field